example/ck_tile/38_block_scale_gemm/README.md

```diff
@@ -7,6 +7,15 @@ This folder contains examples of quant GEMMs using the ck_tile tile-programming
 - Row and Column-wise scaled: All of the rowwise elements in A Matrix and columwise elements in B Matrix will share the same quantization element and the elementwisde operation will complete in epilogue.
 - Tensor-wise scaled: Share the same scalar scale across the whole tensor of A or B
```

Comment:
```suggestion
- Row and Column-wise scaled: All of the row-wise elements in A Matrix and column-wise elements in B Matrix will share the same quantization element and the element-wise operation will complete in epilogue.
- Tensor-wise scaled: Share the same scalar scale across the whole tensor of A or B
```

Reply:
@AviralGoelAMD Can you incorporate the above changes while you are working on this? Thanks